raphtory/src/db/api/view/layer.rs

```diff
@@ -97,6 +101,10 @@ impl<'graph, V: OneHopFilter<'graph> + 'graph> LayerOps<'graph> for V {
         let ids = self.current_filter().valid_layer_ids_from_names(layers);
         self.one_hop_filtered(LayeredGraph::new(self.current_filter().clone(), ids))
     }
+
+    fn num_layers(&self) -> usize {
+        self.current_filter().unique_layers().count()
```

Comment:
Can you make a note of this function in the PR so I can add it to the patch notes